src/AutorestSwift/Models/Complex/ParameterType.swift

```diff
@@ -113,6 +113,10 @@ enum ParameterType: Codable {
         return common.nullable ?? false
     }
 
+    var isConstantSchema: Bool {
+        return common.schema as? ConstantSchema != nil
+    }
+
```

Comment:
It would seem there should be a more generic way to handle this. You should already be able to say

```swift
if property.schema is ConstantSchema {
...
}
```

Otherwise I feel this should be something where you pass in the type you want to compare it to so we don't have a bunch of `isConstantSchema`, `isDateSchema` etc.

Reply:
Good point.